docs/faq/terminology.rst

```diff
@@ -0,0 +1,33 @@
+.. _Observation_Terminology:
+
+***************
+Terminology
+***************
+
+Scheduling
+==================
```

Comment:
Super minor, but it's generally best-practice to have as many `=` characters as the text just above.  So thie would be:
```
Scheduling
==========
```

and similar for the `*`s above (around "Terminology")